gmaps/models.py

```diff
@@ -151,6 +139,28 @@ def delete_all_periodic_task_for_task(sender, instance, **kwargs):
         PeriodicTask.objects.get(name=instance.place.value).delete()
     except PeriodicTask.DoesNotExist:
         pass
+        #TODO add sending error to log
```

Comment:
proponowałbym to od razu przerobić na zadanie w jirze, a tu sie odwałoć, np. "TODO Scrum-99"